### DISCUSSION TITLE
KUKA kr10 R1100-2 of DH parameters

### DISCUSSION THREAD

**Discussion by cr7fzy** [visose/Robots#128] [Q&A] [answered]
Hi, I'm come from Xi'an Jiaotong-Liverpool University,our laboratory has acquired a robotic arm(kr10 r1100-2). I can't find 3dm and xml file in the library of it. So I'm trying to writting about xml file about it, but I can't work out the DH parameters for this robot, so I'm stuck here. Can somebody help me with it asasp. Thanks a lot.

**Comment by visose** [comment, marked as answer]
You can search the existing libraries using the GitHub search on the top. The the `Taiwan.xml` file seems to have this robot: https://github.com/visose/Robots/search?q=KUKA+kr10

The DH parameters represent offsets between joints (distances). You can check by comparing the values between the xml file above and the dimensions in the drawing of this [spec sheet](https://www.kuka.com/-/media/kuka-downloads/imported/6b77eecacfe542d3b736af377562ecaa/0000210360_en.pdf).
